dozer-ingestion/aerospike/src/connector.rs

```diff
@@ -245,28 +245,26 @@ async fn ingestor_loop(
     ingestor: Ingestor,
     operation_id_sender: mpsc::UnboundedSender<PendingOperationId>,
 ) {
-    let mut operation_id = 0;
     while let Some(message) = message_receiver.recv().await {
-        let pending_operation_id = PendingOperationId {
-            operation_id,
-            sender: message.sender,
-        };
-
-        // Propagate panic in the pipeline event processor loop.
-        operation_id_sender.send(pending_operation_id).unwrap();
-
         // Ignore the error, because the server can be down.
         for message in message.messages {
-            let _ = ingestor.handle_message(message).await;
+            let _ = ingestor.handle_message(message).await.unwrap();
```

Comment:
If we `unwrap`, we don't have to write `let _ = `. And the comment above should be removed.